example/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "~16.7.0-alpha.2",
-    "react-dom": "~16.7.0-alpha.2",
-    "react-scripts": "2.1.1",
-    "reinspect": "~0.4.2"
+    "react": "~16.8.2",
```

Comment:
Would it make more sense to make these `peerDependencies`, that way the including lib can use whatever actual version it wants?